api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java

```diff
@@ -318,11 +344,16 @@ public CohortDefinition getTxCurrCompositionCohort(String cohortName, CohortDefi
 		    new Mapped<CohortDefinition>(patientsWithNextConsultationDate, ParameterizableUtil
 		            .createParameterMappings("onOrBefore=${onOrBefore},location=${location}")));
 		
+		TxCurrComposition.getSearches().put(
+		    "13",
+		    new Mapped<CohortDefinition>(patientsEnrrolledOnArtWithoutAnyConsultation, ParameterizableUtil
+		            .createParameterMappings("location=${location}")));
+		
 		String compositionString;
 		if (currentSpec) {
-			compositionString = "(1 OR 2 OR 3 OR 4) AND (NOT (5 OR ((6 OR (NOT 11)) AND (NOT (7 OR 8))))) AND (11 OR 12)";
+			compositionString = "(1 OR 2 OR 3 OR 4) AND (NOT (5 OR ((6 OR (NOT 11)) AND (NOT (7 OR 8))))) AND (11 OR 12) AND NOT 13";
 		} else {
-			compositionString = "(1 OR 2 OR 3 OR 4) AND (NOT (5 OR (6 AND (NOT (7 OR 8)))))";
+			compositionString = "(1 OR 2 OR 3 OR 4) AND (NOT (5 OR (6 AND (NOT (7 OR 8))))) AND NOT 13";
```

Comment:
@Guimino Can this maybe be excluded from the queries in the main union part of the composition by updating/fixing those queries? So that we can reduce the size of the starting set of patients instead of adding a new condition.